tools/generator/Java.Interop.Tools.Generator.CodeGeneration/CodeGenerator.cs

```diff
@@ -362,16 +362,16 @@ internal virtual void WriteConstructor (Ctor constructor, string indent, bool us
 			if (constructor.Annotation != null)
 				writer.WriteLine ("{0}{1}", indent, constructor.Annotation);
 
-			writer.WriteLine ("{0}{1} unsafe {2} ({3})\n{0}\t: {4} (IntPtr.Zero, JniHandleOwnership.DoNotTransfer)",
-					indent, constructor.Visibility, constructor.Name, GenBase.GetSignature (constructor, opt), useBase ? "base" : "this");
+			writer.WriteLine ("{0}{1} unsafe {2} ({3}){5}{0}\t: {4} (IntPtr.Zero, JniHandleOwnership.DoNotTransfer)",
```

Comment:
This and the next line feel similar: use separate `writer.WriteLine()` invocations for each line.

We may also want to consider moving to C#6 interpolated strings as well.

Reply:
Added the `WriteLine()`'s.  Agreed with interpolated strings in our future, but it feels like a "whole-file" change rather than doing a few lines.